src/tools/compiletest/src/header.rs

```diff
@@ -488,6 +493,8 @@ impl TestProps {
                     COMPARE_OUTPUT_LINES_BY_SUBSET,
                     &mut self.compare_output_lines_by_subset,
                 );
+
+                config.set_name_directive(ln, LLVM_COV_USE_COLOR, &mut self.llvm_cov_use_color);
```

Comment:
Given that this is only necessary for one test, would it be better to have a `// llvm-cov-args:` that you can use for one-off cases like this?

Reply:
Done. The main reason I chose `llvm-cov-use-color` is that adding a boolean directive was slightly easier, but switching over to the more general flags directive wasn't too bad after all.